js/scripts.js

```diff
@@ -1,3 +1,15 @@
-var name = prompt('Enter your name');
-alert('Hello, ' + name);
-console.log('Hello, ' + name);
+
+var list = document.getElementById('list');
+var add = document.getElementById('button');
+addEventListener('click', function() {
+    var element = document.createElement('li');
+    var newElem = document.createElement('Item 1');
+    var newElem = document.createElement('Item 2');
+    var newElem = document.createElement('Item 3');
+element.innerHTML = 'Item';
+newElem.innerHTML = 'Item 1';
+newElem.innerHTML = 'Item 2';
+newElem.innerHTML = 'Item 2';
+list.appendChild(element);
+});
+// aż wstyd mi pokazywać ten kod, ale kompletnie nie wiem w ktora strone z tym pojsc :/
```

Comment:
zacznij od przyswojenia co zwraca Ci https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById czy https://developer.mozilla.org/en/docs/Web/API/Document/getElementsByClassName bo to niezbędne do zrozumienia tego co próbujesz zrobić. Ty w tym momencie tworzysz sztucznie jakieś `Item1` `Item2` itp a te wartości powinny być wyciągnięte z HTMLa. Even listener rejestrujesz na jakimś elemencie a u Ciebie (4 linijka) tak się nie dzieje.